site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "minikube start"
+linkTitle: "Get Started!"
+weight: 1
+---
+
+This Tutorial explain how to setup minikube in china.
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
+
+### Binary download
+
+For more detail please refer [minikube installation guide]({{<ref "/docs/start">}})
+
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">2</strong></span>Start your cluster</h2>
+
+From a terminal with administrator access (but not logged in as root), run:
```

Comment:
I would make it clear that this is OS dependent:

```
Open a terminal. If on Windows, open a terminal with administrator access. Run:
```

Reply:
updated..